waku/waku_api/jsonrpc/store/handlers.nim

```diff
@@ -27,16 +28,16 @@ proc toPagingInfo*(pagingOptions: StorePagingOptions): PagingInfoRPC =
   PagingInfoRPC(
     pageSize: some(pagingOptions.pageSize),
     cursor: pagingOptions.cursor,
-    direction: if pagingOptions.forward: some(PagingDirectionRPC.FORWARD)
-               else: some(PagingDirectionRPC.BACKWARD)
+    direction: if pagingOptions.forward: some(PagingDirection.FORWARD)
+               else: some(PagingDirection.BACKWARD)
```

Comment:
Good morning!
Is this conversion needed?

Reply:
I don't think so, I likely missed it because of tunnel vision when updating stuff. Good catch

Reply:
Nevermind, that was actually well done. `pageOptions.forward` is a bool 😂